cookiecutter-xblock/cookiecutter.json

```diff
@@ -1,9 +1,10 @@
 {
   "project_desc": "My First XBlock",
-  "package_name": "myxblock",
+  "package_name": "my_xblock",
+  "i18n_namespace": "{{cookiecutter.package_name.split('_')|map('capitalize')|join('')}}I18n",
   "github_org": "openedx",
-  "repo_name": "{{cookiecutter.package_name}}-xblock",
-  "tag_name": "myxblock",
+  "repo_name": "{{cookiecutter.package_name.replace('_', '-')}}",
```

Comment:
it is intentional, I see it as more convenient to have the repo name the same as the package name unless the creator decides otherwise. No technical reason or implication for that

Note: the template will not **_enforce_** anything. It'll just propose these as defaults, and the XBlock creator can change them on the spot

Reply:
another thing is that we have a new field `i18n_namespace` that will look nicer by this change